src/generic/datavgen.cpp

```diff
@@ -2645,6 +2838,20 @@ void wxDataViewMainWindow::OnPaint( wxPaintEvent &WXUNUSED(event) )
 
         cell_rect.x += cell_rect.width;
     }
+
+#if wxUSE_DRAG_AND_DROP
+    if (m_dropItemInfo.m_hint == DropHint_Below || m_dropItemInfo.m_hint == DropHint_Above)
+    {
+        const int insertLineHeight = 2;     // TODO: setup (should be even)
```

Comment:
Should this use `FromDIP(2)` or do you really want to use 2 physical pixels even on high DPI displays?

Reply:
I am familiar with wx's HiDPI.
I suppose It should be 2 physical pixels here (1px from the row above and 1px from the row below)